packages/skeleton-react/package.json

```diff
@@ -6,7 +6,8 @@
 		"dev": "vite",
 		"build": "tsc && vite build",
 		"lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
-		"preview": "vite preview"
+		"preview": "vite preview",
+		"test": "vitest"
```

Comment:
I suggest using the command `vitest run` if you plan on using this command in a CI environment. Just the `vitest` command starts the testing in watch mode which doesn't terminate a pipeline.